crates/forge-runner/src/build_trace_data.rs

```diff
@@ -148,7 +144,11 @@ pub fn build_profiler_execution_resources(
         vm_resources: VmExecutionResources {
             n_steps: execution_resources.n_steps,
             n_memory_holes: execution_resources.n_memory_holes,
-            builtin_instance_counter: execution_resources.builtin_instance_counter,
+            builtin_instance_counter: execution_resources
+                .builtin_instance_counter
+                .into_iter()
+                .map(|(key, value)| (key.to_str_with_suffix().to_owned(), value))
+                .collect(),
```

Comment:
Why this change?

Reply:
`.collect()` change form `.collect_vec()`. Better type inference